config/database.yml

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
- [ ] Please make sure that the app works properly. Currently, when I clicked the "Click me" button, I got the following error. 

![image](https://user-images.githubusercontent.com/62290545/236628945-246f8281-40da-4b8b-87c7-7c76845c3a48.png)